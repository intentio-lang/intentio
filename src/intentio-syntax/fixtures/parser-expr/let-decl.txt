let e = r#"my sweet "string""#
