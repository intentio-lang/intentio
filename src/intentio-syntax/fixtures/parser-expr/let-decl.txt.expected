{
  "_letDeclName": "e",
  "_letDeclVal": {
    "contents": {
      "contents": "r#\"my sweet \"string\"\"#",
      "tag": "RawString"
    },
    "tag": "LitExpr"
  },
  "tag": "LetDeclExpr"
}
